cmake/compilers/Intel.cmake

```diff
@@ -26,10 +26,9 @@ elseif (APPLE)
     set(TBB_IPO_COMPILE_FLAGS $<$<NOT:$<CONFIG:Debug>>:-ipo>)
 else()
     include(${CMAKE_CURRENT_LIST_DIR}/GNU.cmake)
-    set(TBB_LIB_LINK_FLAGS ${TBB_LIB_LINK_FLAGS} -static-intel -Wl,-z,relro,-z,now,)
-    set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -fstack-protector -Wformat -Wformat-security
-                                 $<$<NOT:$<CONFIG:Debug>>:-qno-opt-report-embed -D_FORTIFY_SOURCE=2>
+    set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} $<$<NOT:$<CONFIG:Debug>>:-qno-opt-report-embed -D_FORTIFY_SOURCE=2>
```

Comment:
Shouldn't `-D_FORTIFY_SOURCE=2` be also used for GNU?

Reply:
-D_FORTIFY_SOURCE=2 fortification requires __OPTIMIZE__ macro greater than 0 which is not present for GNU